src/app/pages/quickMaps/pages/baselineDetails/chartCard/chartCard.component.scss

```diff
@@ -11,21 +11,16 @@ ngx-chartjs {
   background-size: cover;
 }
 
-.example-spacer {
-  flex: 1 1 auto;
-  text-align: center;
-  font-size: 17px;
-}
-
 .tab-element {
   margin: 5px;
 }
 
-table {
+::ng-deep table {
```

Comment:
Shouldn't use ::ng-deep unless it's nested at least one level, other wise it will affect every table everywhere.